W-23478329 Add scope on permission error (#63)
mulesoft/docs-mulesoft-mcp-server: modules/ROOT/pages/getting-started-platform.adoc

* If you encounter an authorization error:
** Add the correct redirect URL to your connected app.
** Add the necessary permissions scopes to your connected app.

> * If you encounter a permissions-related error when attempting to authorize, add the following permission scopes to your connected app: 

Reviewer: ```suggestion
* If you encounter a permissions-related error when attempting to authorize, add these permission scopes to your connected app: 
```
